util/quickstart/setchplenv.bash

```diff
@@ -58,8 +58,8 @@ export CHPL_TASKS=fifo
 echo "Setting CHPL_MEM to cstdlib"
 export CHPL_MEM=cstdlib
 
-echo "Setting CHPL_GMP to none"
-export CHPL_GMP=none
+echo "Setting CHPL_GMP to system"
+export CHPL_GMP=system
```

Comment:
I don't think this change was meant to be in this PR.

Reply:
Ohh, Actually it was for my local system to use `BigInt`  library.